website/docs/API.md

```diff
@@ -193,6 +202,24 @@ optional message
 </View>
 ```
 
+You can use the mapProps option to transform the props that will be printed : 
+
+```jsx
+render(<View style={{backgroundColor: 'red'}}/>);
+debug({ mapProps : ({style, ...props}) => ({props}) })
+```
+
+This will log the rendered JSX without the `style` props. 
+
+This option can be used to target specfic props when debugging a query (for instance keeping only children prop when debugging a byText query) and you can also transform prop values so that they are more readable (e.g. flatten styles).
```

Comment:
Not sure about mentioning `children` prop in this case, as it is treated specially by `mapProps`

```suggestion
This option can be used to target specific props when debugging a query (for instance keeping only `children` prop when debugging a `getByText` query) and you can also transform prop values so that they are more readable (e.g. flatten styles).
```

Reply:
I changed the order so that it is explained first that children prop cannot be filtered then give the example of usage, since I believe this will be one of the most common, at least until byRole queries become more popular